numba_dpex/experimental/models.py

```diff
@@ -6,14 +6,28 @@
 numba_dpex.experimental module.
 """
 
+from llvmlite import ir as llvmir
 from numba.core.datamodel import DataModelManager, models
+from numba.core.datamodel.models import PrimitiveModel
 from numba.core.extending import register_model
 
 import numba_dpex.core.datamodel.models as dpex_core_models
 
+from .literal_intenum_type import IntEnumLiteral
 from .types import KernelDispatcherType
 
 
+class LiteralIntEnumModel(PrimitiveModel):
```

Comment:
Should we be consistent with names? `<class>` and `<class>Model` sounds better for me.

Reply:
Done